sdk/vision/ci.yml

```diff
@@ -35,11 +35,20 @@ parameters:
     type: boolean
     default: true
 
+# Note that EnableVerifyTypeSpecCodeGeneration is set to false here in order to skip
+# the task "Verify TypeSpec Code Generation" in the pipeline. This task is currenlty
+# failing because the tools do not account for the additional code snippet that were
+# inserted into Java source code comments (JavaDocs). This task will be enabled again
+# once the tools are updated. During development of the next release of the SDK, please
+# enable this check temporarly in your PRs and make sure the only difference you see
+# when the "Verify TypeSpec Code Generation" task fails, is because of code snippets
+# in source code comments (JavaDocs).
```

Comment:
File a GitHub issue to track this work item and link this to the backlog issue in AutoRest Java repo. So, when the codegen changes are ready, we will know that this task is unblocked. 

Also, another thing we can do (in a separate PR) is to not skip this step entirely but to not fail the build on error instead (by using [ContinueOnError](https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-task?view=azure-pipelines)). So, it won't block merging PR but it will also remind us that there are TypeSpec changes that don't match which can prompt devs to look and validate if it's just code snippets or if there's more.

Reply:
Codegen issue on https://github.com/Azure/autorest.java/issues/2492

Reply:
Opened https://github.com/Azure/azure-sdk-for-java/issues/38337 . Thanks!